lib/cloudflare/accounts.rb

```diff
@@ -36,9 +46,5 @@ def representation
 			Account
 		end
 
-		def create(name)
```

Comment:
I found out that this isn't a supported action for accounts so I removed it.

Reply:
Awesome.